checks/evaluation/pinned_dependencies.go

```diff
@@ -26,19 +26,19 @@ import (
 
 var errInvalidValue = errors.New("invalid value")
 
-type pinnedResult int
+type PinnedResult int
 
 const (
-	pinnedUndefined pinnedResult = iota
-	pinned
-	notPinned
+	PinnedUndefined PinnedResult = iota
+	Pinned
+	NotPinned
 )
 
 // Structure to host information about pinned github
 // or third party dependencies.
-type worklowPinningResult struct {
-	thirdParties pinnedResult
-	gitHubOwned  pinnedResult
+type WorkflowPinningResult struct {
+	ThirdParties PinnedResult
+	GitHubOwned  PinnedResult
```

Comment:
These need not be exposed. `evaluation` package is not something the attestor should depend on. It should work on the raw data directly - https://github.com/ossf/scorecard/blob/main/checker/raw_result.go#L26.